.ci/oci-e2e-deployment.sh

```diff
@@ -48,6 +50,12 @@ function catchFinish() {
     exit $JOB_EXIT_CODE
 }
 
+function installCITools() {
+
+    wget --header 'Authorization: token $GITHUB_TOKEN' https://github.com/mikefarah/yq/releases/download/v4.20.2/yq_linux_amd64 -O /tmp/bin/yq
```

Comment:
why GITHUB_TOKEN?

Reply:
@Michkov 
The ci throw this error when trying to install yq:
```log
Resolving github.com (github.com)... 140.82.114.3
Connecting to github.com (github.com)|140.82.114.3|:[44](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/redhat-appstudio_e2e-tests/112/pull-ci-redhat-appstudio-e2e-tests-main-redhat-appstudio-e2e/1536972675975483392#1:build-log.txt%3A44)3... connected.
HTTP request sent, awaiting response... 429 Too Many Requests
2022-06-15 07:24:32 ERROR 429: Too Many Requests.
```